### DISCUSSION TITLE
Facing issue in connection with Databricks

### DISCUSSION THREAD

**Discussion by vkgyl** [getredash/redash#6383] [Q&A]
Hi all, 
can anyone help me getting redash connected with databricks? 
I am facing the below issue - 

`Connection Test Failed:
MALFORMED_REQUEST: Client Python/THttpClient (gunicorn) is not supported for SQL warehouses.SimbaJDBCDriver 2.6.14, SimbaODBCDriver 2.6.15 or above is required.`

i am hosting open source redash on AWS using the AMI given by redash - ami-0ff23052091536db2 
redash version - 8.0.0+b32245

**Comment by justinclift** [comment]
Ouch.  There *might* not be a good solution for this any time soon (weeks at least).  The newest official release of Redash is version 10.1, and that was about two years ago.

We're looking to make a follow up Redash release in the next few weeks, which probably *would* work for this... but we haven't really thought through the upgrade path from older versions yet.

**Reply by maximStafeevAtAllscripts** [reply]
Hi!! I also trying to connect to Databricks. Got the same error on redash:latest, and cannot connect from both :10.1.0.b50633 and :preview. What can you suggest me?

**Reply by marinaolina** [reply]
@maximStafeevAtAllscripts I would suggest to build your own docker image following[ manual](https://github.com/getredash/redash/wiki/Local-development-setup)
It worked for me fine and  databricks connection is restored

**Comment by vkgyl** [comment]
Thanks Justin for answering and giving update on the upcoming release.

Is there a way i can use the latest official release version 10.1 on aws cloud by any official AMI or i will have to set it up myself?

**Reply by justinclift** [reply]
How good are you with the Linux command line?  If you're comfortable using ssh and things like bash, then you'll be ok.

If not though, it's probably not a good idea. :smile:

**Reply by vkgyl** [reply]
thanks justing for letting me know.
i am comfortable with linux command line. Is there some documentation where i can refer to implement this on my own?

**Reply by justinclift** [reply]
Cool. :smile:

In that case, the upgrade process itself that's documented here is what you'll need:

https://redash.io/help/open-source/admin-guide/how-to-upgrade

Before doing that, to be safe *please* make a backup of your database and the matching environment file (probably `/opt/redash/env`).  That environment file contains the encryption key used for the database, so don't miss it. :smile:

The page there lists docker images (eg `redash/redash:8.0.0.b32245`) that you can use.  It doesn't list the more recent ones though.

This is the actual, complete list:

https://hub.docker.com/r/redash/redash/tags

You'll probably want the `10.1.0.b50633` one for now.  So, that would become `redash/redash:10.1.0.b50633` in the `docker-compose.yml` image.

Try upgrading to that, and see how you go. :smile:

**Reply by vkgyl** [reply]
Thanks @justinclift. i was able to upgrade the redash version to 10.1.0 following these instructions.
Now i met with error mentioned in #6176 but not able to get this resolved using workaround too.
Any solution that you would recommend which can work?

Thanks again for being very helpful.

**Reply by justinclift** [reply]
Ouch, that's not good.  Um, I'm due to try building some initial "alpha quality" docker images from our latest source code (prior to our first "Community" release).  Was going to get that done today, but it might be a tomorrow thing instead.

Um... are you ok to wait maybe a day and be an early test bunny for those?

**Reply by vkgyl** [reply]
Sure thing Justin, i am eagerly waiting for the image to try out and getting my problem resolved.

**Reply by vkgyl** [reply]
hi @justinclift did you get the chance to release the alpha version for images? if yes, can you help me how do i access these?

**Reply by justinclift** [reply]
Oops.  Got very distracted by other stuff.  Um, give me a few hours and I'll get it done. :smile:

**Reply by justinclift** [reply]
Ugh.  Looks like I don't have time to get this done today after all.  Sorry @vkgyl, it'll have to be tomorrow. :frowning:
